src/frontend/shared-components/src/confirmation-popup/ConfirmationPopup.js

```diff
@@ -1,16 +1,16 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
why the comma?

Reply:
you need it because they are different modules you are importing
```
import moduleA, moduleB, { moduleC } from "somepackage" 
```

Reply:
can't read, they look so different with formating. sorry.

Reply:
List of things being imported from react